kie-drools-wb/kie-drools-wb-distribution-wars/src/main/productized/webapp/logout.jsp

```diff
@@ -5,10 +5,13 @@
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ taglib uri="http://jakarta.apache.org/taglibs/i18n-1.0" prefix="i18n" %>
 <%
-  request.logout();
-  javax.servlet.http.HttpSession httpSession = request.getSession(false);
-  if (httpSession != null) {
-    httpSession.invalidate();
+  try {
+    request.logout();
+    javax.servlet.http.HttpSession httpSession = request.getSession(false);
+    if (httpSession != null) {
+      httpSession.invalidate();
+    }
+  } catch ( Exception e ) {
```

Comment:
I try to avoid use comments on code as much as possible. In my opinion, the swallow here is `clear` as a fail would create no side effect.
Do you think that a WARN log would be enough?